app/serializers/doorkeeper/token_serializer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Doorkeeper
+  class TokenSerializer
+    include JSONAPI::Serializer
+    attributes :token, :token_type, :expires_in, :refresh_token
```

Comment:
we can have a line between `attributes` and `attribute`, as we provide a block to it and remaining are just single line

Reply:
Done ;-)